### PR TITLE
Clean wallet connections

### PR DESCRIPTION
**Pull request by ronaldsg20**
[TWPAPP-995](https://rsklabs.atlassian.net/jira/software/projects/TWPAPP/boards/128?selectedIssue=TWPAPP-995)

### CODE REVIEW

File: src/pegout/components/PegoutForm.vue [hunk 1]
```diff
@@ -294,6 +294,14 @@ export default defineComponent({
       showTxErrorDialog.value = true;
     }
 
+    const valueToReceive = computed<WeiBig>(() => {
+      const quoteHash = selectedQuoteHash.value || '';
+      if (quoteHash) {
+        return flyoverPegoutState.value.amountToTransfer;
+      }
+      return pegOutTxState.value.amountToTransfer;
```
Comment (lserra-iov) [inline src/pegout/components/PegoutForm.vue:302]: We have to take into account the fees here

Comment (annipi) [reply src/pegout/components/PegoutForm.vue]: I just updated the value on the nativeQuote 👌

**Review by alexjavabraz — APPROVED** [review]
LGTM

File: src/common/router/index.ts
Comment (lserra-iov) [inline src/common/router/index.ts:1]: confirmations prop is missing here

Comment (annipi) [reply src/common/router/index.ts:1]: Good catch!

**Review by lserra-iov — APPROVED** [review]

**Review by leoiovlabs — APPROVED** [review]
LGTM

**Pull request merged**